apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx

```diff
@@ -47,7 +46,10 @@ export const useMultipleBannerConfig = () => {
         shouldRender: isRenderUserBanner.shouldRender && !isRenderUserBanner.isEarningsBusdZero,
         banner: <UserBanner />,
       },
-      { shouldRender: shouldRenderIfoBanner, banner: <ListaIFOBanner /> },
```

Comment:
Don't delete the `ListaIFOBanner` just list under it.

Reply:
@chefjackson say Lista can delete now

Reply:
Can u comment out the code `shouldRenderIfoBanner` in line  `17`,`41`,`101`

Reply:
done